src/test/java/org/cactoos/io/TempFileTest.java

```diff
@@ -106,7 +106,7 @@ public void createFileWithSuffix() throws Exception {
             "randomSuffix%s", System.currentTimeMillis()
         ).asString();
         try (TempFile file = new TempFile("", suffix)) {
-            MatcherAssert.assertThat(
+            new Assertion<>(
                 "File not created with the given suffix",
                 file.value().getFileName().toString(),
                 Matchers.endsWith(suffix)
```

Comment:
@marceloamadeu can you change that to the object oriented version coming from cactoos-matchers?

Reply:
@victornoel Done ... please check.